### DISCUSSION TITLE
SEER processing

### DISCUSSION THREAD

**Discussion by hminle** [Jeanselme/NeuralFineGray#6] [Q&A] [answered]
Hi Vincent,

In this file, I do not understand what you do with `df_ord`
![image](https://github.com/Jeanselme/NeuralFineGray/assets/16201681/0bbe7ecf-fc41-41f1-ad13-33019592864e)

Look like you process it, but then you do not use it after that.

Thanks

**Comment by Jeanselme** [comment, marked as answer]
You are absolutely right. This is a mistake, let me change this ! Thank you for catching it
